<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shv::visu::logview::DataViewWidget</class>
 <widget class="QWidget" name="shv::visu::logview::DataViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>54</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="pbShowRawData">
     <property name="toolTip">
      <string>Show/hide raw data</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../images/images.qrc">
       <normaloff>:/shv/visu/images/raw.svg</normaloff>:/shv/visu/images/raw.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="pbShowChannelFilterDialog">
     <property name="toolTip">
      <string>Show filter dialog</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../images/images.qrc">
       <normaloff>:/shv/visu/images/filter-off.svg</normaloff>:/shv/visu/images/filter-off.svg</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
